.docker/images/govcms8/settings/settings.php

```diff
@@ -195,6 +207,10 @@ public function __toString() {
       ],
     ];
   }
+  catch (\Exception $error) {
+    $settings['container_yamls'][] = 'sites/default/redis-unavailable.services.yml';
```

Comment:
This doesn't match the file name which has a typo: "redis-unavaiable.services.yml" (although that might not be a bug cos I can't see how it's being copied in from here).

Reply:
Nice! Thanks :)